docs/source/guides/configuration.rst

```diff
@@ -56,6 +59,21 @@ Environment-specific parameters
 - ``mst_enable`` enables or disables multisignature transaction support in
   Iroha. We recommend setting this parameter to ``false`` at the moment until
   you really need it.
+- ``mst_expiration_time`` the time period after which a not fully signed MST
+  is considered expired.
+- ``max_rounds_delay`` optional parameter specifying the maximum delay between
+  two consensus rounds.
+  When Iroha is staying idle, it gradually increases the delay to reduce CPU,
+  network and logging load.
+  However too long delay may be unwanted when first transactions arrive after a
+  long idle time.
+  This parameter allows users to find an optimal tradeoff.
```

Comment:
> tradeoff — a balancing of factors all of which are not attainable at the same time

allows users to find an optimal value in a tradeoff between resource consumption and transaction throughput rate.